src/app/game-state/item-repo.service.ts

```diff
@@ -625,39 +625,53 @@ export class ItemRepoService {
       value: 9,
       description: 'A spiritual quality log.',
     },
+    redwoodLog: {
```

Comment:
Redwood is pretty soft. Seems like this should be a lower cost wood. Let's give these unique descriptions too.

Reply:
Oop, forgot descriptions. Not sure where to place the woods, because some are useful for structures and others have a way different janka scale than what's already here.

Reply:
Kinda want to make elm balsa instead.

Reply:
Alright, roughed some locations based on softness since harder woods are more suitable for weapons, existing woods were really wherever they wanted to be.